src/umpire/CMakeLists.txt

```diff
@@ -84,13 +84,22 @@ set (umpire_depends
   umpire_util
   umpire_tpl_judy)
 
+if (UMPIRE_ENABLE_HIP)
+  set (umpire_depends
+    ${umpire_depends}
+    blt::hip)
+endif()
+
 if (UMPIRE_ENABLE_OPENMP_TARGET)
   set (umpire_depends
     ${umpire_depends}
     openmp)
 endif ()
 
 blt_list_append( TO umpire_depends ELEMENTS umpire_interface IF UMPIRE_ENABLE_C )
+blt_list_append( TO umpire_depends ELEMENTS umpire_device IF UMPIRE_ENABLE_CUDA)
```

Comment:
These two lines should be removed.

Reply:
Do you mean lines 99 and 100 or 100 and 101?